notebooks/machine_learning/raw/ex6.ipynb

```diff
@@ -129,22 +112,48 @@
    ]
   },
   {
-   "cell_type": "markdown",
+   "cell_type": "code",
+   "execution_count": null,
    "metadata": {},
+   "outputs": [],
    "source": [
-    "# Think about Your Results\n",
-    "Under what circumstances might you prefer the Decision Tree to the Random Forest, even though the Random Forest generally gives more accurate predictions?  Weigh in or follow the discussion in [this discussion thread](https://www.kaggle.com/learn-forum/60582).\n",
+    "#%%RM_IF(PROD)%%\n",
     "\n",
-    "# Keep Going\n",
+    "from sklearn.ensemble import RandomForestRegressor\n",
     "\n",
-    "So far, you have followed specific instructions at each step of your project. This helped learn key ideas and build your first model, but now you know enough to try things on your own. \n",
+    "# Define the model. Set random_state to 1\n",
+    "rf_model = RandomForestRegressor(random_state=1)\n",
+    "\n",
+    "# fit your model\n",
+    "rf_model.fit(train_X, train_y)\n",
     "\n",
-    "Machine Learning competitions are a great way to try your own ideas and learn more as you independently navigate a machine learning project.  Learn **[how to submit your work to a Kaggle competition](https://www.kaggle.com/dansbecker/submitting-from-a-kernel)**.\n",
```

Comment:
Up to you, but I think in a case like this it'd be clearer to keep the old outro format, and just replace the hardcoded url with a macro. I wouldn't feel compelled to use the `KEEP_GOING` macro at the end of every exercise notebook.